ouroboros-consensus-cardano/app/DBAnalyser/Parsers.hs

```diff
@@ -129,24 +129,35 @@ storeLedgerParser = do
     (  long "store-ledger"
     <> metavar "SLOT_NUMBER"
     <> help "Store ledger state at specific slot number" )
-  ledgerValidation <- flag LedgerReapply LedgerApply
-    (  long "full-ledger-validation"
-    <> help (  "Use full block application while applying blocks to ledger states, "
-            <> "also validating signatures and scripts. "
-            <> "This is much slower than block reapplication (the default)."
-            )
-    )
+  opts <- pApplyOpts
   doChecksum <- flag DoDiskSnapshotChecksum NoDoDiskSnapshotChecksum
     (mconcat [ long "no-snapshot-checksum-on-write"
              , help (unlines [ "Don't calculate the checksum and"
                              , "write the '.checksum' file"
                              , "when taking a ledger snapshot"
                              ])
              ])
-  pure $ StoreLedgerStateAt slot ledgerValidation doChecksum
+  pure $ StoreLedgerStateAt slot opts doChecksum
+
+pApplyOpts :: Parser ApplyOpts
+pApplyOpts = do
+  val <- flag ValidationOn ValidationOff
+    (  long "full-ledger-validation"
+    <> help (  "Use full block application while applying blocks to ledger states, "
+            <> "also validating signatures and scripts. "
+            <> "This is much slower than block reapplication (the default)."
+            )
+    )
+  evs <- flag ComputeLedgerEvents OmitLedgerEvents
+    (  long "compute-ledger-events"
+    <> help (  "Although the events will not be emitted anywhere (yet), "
```

Comment:
Since `db-analyser` is often about performance, I think it'd help if you disambiguated "compute such events". Right now (due to the "yet"), the ledger would merely allocate thunks for the events, but nothing would ever force those thunks, right?

Reply:
I undid these changes